iso_templates/isolinux.cfg.template

```diff
@@ -6,5 +6,5 @@ LABEL clear
   MENU DEFAULT
   MENU LABEL Clear Linux OS for Intel Architecture
```

Comment:
Please change to:
"Clear Linux* OS"

Reply:
Done, updated all other instances I could find as well.